### ISSUE TITLE
Alternatives to .route()

### ISSUE THREAD

**Issue by slothstronaut** [ionic-team/ionic-plugin-deeplinks#221]
The plugin is working great, but I had to dig into the source to find what I needed, which was actually just to get the whole URL back.

It would be great if you could add some examples that are more generic, for example all I needed was:

```
 IonicDeeplink.onDeepLink((data) => {
            console.log(data);            
 });
```

**Comment by chuckdries** [comment]
Thank you! This is going to make integrating with React Router much easier

**Comment by thomasmodeneis** [comment]
Is there any way to integrate with react ?

**Comment by chuckdries** [comment]
> Is there any way to integrate with react ?

Yeah, use react-router and call `history.push()` in the onDeepLink callback
